packages/medusa/src/repositories/price-list.ts

```diff
@@ -2,4 +2,26 @@ import { EntityRepository, Repository } from "typeorm"
 import { PriceList } from "../models/price-list"
 
 @EntityRepository(PriceList)
-export class PriceListRepository extends Repository<PriceList> {}
+export class PriceListRepository extends Repository<PriceList> {
+  async listAndCount(query, groups): Promise<[PriceList[], number]> {
+    const qb = this.createQueryBuilder("price_list")
+      .where(query.where)
+      .skip(query.skip)
+      .take(query.take)
+
+    if (groups) {
```

Comment:
the groups argument is not marked as optional? I let you see that as part of the typings above